helpers/sortByMapping.js

```diff
@@ -7,34 +7,51 @@ module.exports = function( dust ) {
 
   /*
   * @description Sorting method that orders a list of things based on a provided map
-  * @param {object} eventsObject Object to sort
-  * @param {string} map Map to use to sort with
-  * @example {@_sortByMapping eventsObject=eventsReply.events map="ASHLKE, ASHWKE, ASHJBE} {/_sortByMapping} output loop of events
+  * @param {object} sortObject - Object to sort
+  * @param {string} map - Map to use to sort with
+  * @param {string} indexes - index structure of response to sort on
+  * @example {@_sortByMapping sortObject=packageRatesReply.packageRates indexes="links, hotelProducts, ids" map="PAUGLA, PAUFOR, PAUELM, PAUFAR"} {/_sortByMapping}
+  * At present their is no unit test coverage for this helper, this is being addressed by changing the structure of our helpers to ensure functions available outside scope of dust
   */
 
   dust.helpers._sortByMapping = function( chunk, context, bodies, params ) {
-    var eventsObject = params.eventsObject;
-    var map = params.map.split(','); // Split string into array
+    // Object to sort
+    var sortObject = params.sortObject;
+    // Mapping to sort to the object to
+    var map = params.map;
```

Comment:
Does this need to be declared? Since you only use the variable once..? Maybe for clarity I guess it is a good idea, but maybe a more descriptive variable name in that case?